multi-transfer-esdt/src/lib.rs

```diff
@@ -8,6 +8,7 @@ use transaction::{EthTransaction, PaymentsVec, Transaction, TxNonce};
 
 const DEFAULT_MAX_TX_BATCH_SIZE: usize = 10;
 const DEFAULT_MAX_TX_BATCH_BLOCK_DURATION: u64 = 100_000_000u64;
+const GAS_LIMIT_ESDT_TRANSFER: u64 = 500_000;
```

Comment:
this will need to be 200_000 to consume less gas (100_000 does not work)

Reply:
Fixed